### PR TITLE
logging: Add possibility to log into file

### PR DESCRIPTION
**Pull request by kosstennbl**
## Description
Add processing of environment variables LOGPATH and LOG_PATH If either one exists - logger will log into the file on that path instead of STDOUT.

## Issues:
Refs: #2000

## How has this been tested:
Manual testing of the functionality.
Running new spec.

## Types of changes:
New non-breaking feature

## Checklist:
**Documentation**
No documentation update is needed.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -47,6 +47,13 @@ crystal src/cnf-testsuite.cr <testname>
 
 ---
 
+### Logging Parameters
+
+- **LOG_LEVEL** environment variable: sets minimal log level to display: error (default); info; debug.
+- **LOG_PATH** environment variable: if set - all logs would be appended to the file defeined by that variable.
```
Comment (taylor) [inline USAGE.md:53]:
```suggestion
- **LOG_PATH** environment variable: if set - all logs would be appended to the file defined by that variable.
```

typo @kosstennbl

Comment (kosstennbl) [reply USAGE.md]: Fixed, thanks

**Review by HashNuke — APPROVED** [review]
LGTM. Will try this out on the pair machine and push to build on github actions.

**Review by martin-mat — APPROVED** [review]
lgtm

**Pull request merged**